core/src/main/java/org/apache/carbondata/core/datastore/page/DecoderBasedFallbackEncoder.java

```diff
@@ -57,10 +57,7 @@ public DecoderBasedFallbackEncoder(EncodedColumnPage encodedColumnPage, int page
     int pageSize =
         encodedColumnPage.getActualPage().getPageSize();
     int offset = 0;
-    int[] reverseInvertedIndex = new int[pageSize];
```

Comment:
What does these changes for？

Reply:
In case of No_Sort where inverted index is not there, this variable was being created unnecessarily.